vitals/validateRange.java

```diff
@@ -0,0 +1,43 @@
+package extensions;
+
+import java.time.temporal.ValueRange;
+
+public class validateRange {

```

Comment:
class name is validateRange but holds checkTemperatureChange method ?
Class validateRange is closed for modification..as we introduce new rangecheck class will become god class